airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -129,10 +136,25 @@ public void run(final ConnectionUpdaterInput connectionUpdaterInput) throws Retr
               StandardSyncSummary standardSyncSummary = standardSyncOutput.get().getStandardSyncSummary();
 
               if (standardSyncSummary != null && standardSyncSummary.getStatus() == ReplicationStatus.FAILED) {
+                failures.addAll(standardSyncOutput.get().getFailures());
+                partialSuccess = standardSyncOutput.get().getStandardSyncSummary().getTotalStats().getRecordsCommitted() > 0;
```

Comment:
Nit: You can use the variable `standardSyncSummary` instead of `standardSyncOutput.get().getStandardSyncSummary()` here.